index.html

```diff
@@ -508,48 +552,58 @@ <h2 class="sign-up-block__title title">Get Start New Job</h2>
         </div>
       </section>
       <!-- Third section -->
-      <section class="third-section">
-        <div class="third-section__background">
-          <div class="container third-section__container">
+      <section class="post-job">
+        <div class="post-job__background">
```

Comment:
```suggestion
      <section class="post-job">
```
Move background to `.post-job`